include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -6874,8 +6901,12 @@ FEEvaluation<dim,
   this->jacobian = &this->mapping_data->jacobians[0][offsets];
   this->J_value  = &this->mapping_data->JxW_values[offsets];
 
-  for (unsigned int i = 0; i < VectorizedArrayType::size(); ++i)
+  unsigned int i = 0;
+  for (; i < this->matrix_free->n_active_entries_per_cell_batch(this->cell);
```

Comment:
I think we might need to see over https://github.com/dealii/dealii/blob/668e626b5edf58149cfb5e6fd74ece09a9a8b6df/include/deal.II/matrix_free/matrix_free.h#L2581-L2589 which implements the `n_active_entries_...` function, in order to limit costs to a single index lookup in these performance-critical places. (For example, `dof_info` stores this number, but we allow to not set it up.)

Reply:
I have created an issue #13263.